src/main/java/org/team1540/robot2018/Tuning.java

```diff
@@ -1,4 +1,201 @@
 package org.team1540.robot2018;
 
+import org.team1540.base.adjustables.Tunable;
+
 public class Tuning {
```

Comment:
Trying to do anything with everything just dumped in SmartDashboard is horrifying difficult. These absolutely should be grouped in some hierarchical fashion. Sendables exist for a reason.

Reply:
Fair, but adding that functionality to the `AdjustableManager` in a sane way is a fairly large undertaking. If you want to not use the manager and just set up `Tuning` to have a `Sendable` table, be my guest, but that's a bunch of uber-boilerplate that I don't want to deal with.

Reply:
Needs to be done in some capacity though, please. Trying to work with this as-is is a total PITB. I'll do this if I have time I guess.